### PR TITLE
Method to check if a video has any age restriction.

### PR DESCRIPTION
**Pull request by jacknguyen**

### CODE REVIEW

File: lib/yt/models/video.rb [hunk 1]
```diff
@@ -247,6 +247,12 @@ def licensed?
         content_detail.licensed_content || false
       end
 
+      # @return [Boolean] whether the video is identified by YouTube as
+      #   age-restricted content.
+      def age_restricted?
+        content_detail.content_rating != nil
```
Comment (claudiofullscreen) [inline lib/yt/models/video.rb]: Hello @jacknguyen and thanks for your contribution! 
The source code of yt is a little hard to grasp at first, and you went right for it! :clap: 

Regarding this line, you are stating that having _any_ value in content rating means that the video is "age-restricted". 

However there are [many different values according to the docs](https://developers.google.com/youtube/v3/docs/videos#contentDetails.contentRating).
For instance, the value might indicate that:

> These programs are intended for children (other than preschool children) who are younger than 14 years of age.

In that case, the video is not really "restricted" to an age, just recommended.
Could you elaborate on a case where you needed to know if there was _any_ content-rating in place, independently of the value of the content rating itself?

Comment (claudiofullscreen) [inline lib/yt/models/video.rb]: Looking back at your https://github.com/Fullscreen/yt/issues/173#issuecomment-104075497 I realize that maybe what you intended to go for is to check if there is any value for `contentDetails.contentRating.ytRating` instead.

That would make sense. If `"contentRating"` is a Hash and the `"ytRating"` key as the `"ytAgeRestricted"` value, then the video is age-restricted. If you change your pull request to check for that instead, I'll accept it :+1: 

If possible, squash your commits too :dancers:

**Comment by jacknguyen** [comment]
Hi @claudiofullscreen.

That was actually also a concern I had with only checking whether contentRating was nil.

I had to include the check for nil as well as ytRating because I kept getting no method error when trying to access the Hash of a nilClass. Made the line a little longer than I would like.

Also I was able to squash the commits, I hope you didn't see my earlier attempts at it :smile:.

**Comment by jacknguyen** [comment]
I'll also update the documentation as well, once you accept. :+1:

**Comment by claudiofullscreen** [comment]
@jacknguyen I've accepted your commit in #198 which also adds https://github.com/Fullscreen/yt/commit/69906de4f98eea30558bb8d7f75cf26d8df4d9a7 to slightly reduce the amount of code and add the CHANGELOG.

Feel free to open a Pull Request against the `gh-pages` branch now if you'd like to update fullscreen.github.io/yt accordingly :grin: 

Thanks for your work!

**Pull request closed without merging**